sonic-sensormond/scripts/sensormond

```diff
@@ -22,6 +23,8 @@ NOT_AVAILABLE     = 'N/A'
 CHASSIS_INFO_KEY  = 'chassis 1'
 INVALID_SLOT      = -1
 
+PLATFORM_ENV_CONF_FILE = "/usr/share/sonic/platform/platform_env.conf"
```

Comment:
This configuration file name is already taken by broadcom so we should probably rename it to something different.
Unless this was thought through and we feel it's fine for a config file to be shared by both asic drivers and generic daemons.
I believ we should CC someone from Broadcom to make sure they're ok with that.
https://github.com/sonic-net/sonic-buildimage/blob/e74ef02efdb3a76803124e961f02ef3f7c756867/platform/broadcom/saibcm-modules/debian/opennsl-modules.init#L98